src/arch/mips/kernel/Mybuild

```diff
@@ -27,11 +27,20 @@ module exception {
 	source "exception.c"
 }
 
+/*
```

Comment:
what's different between previous and new version of module interrupt describing?

Reply:
I thought irq api and irq control api was not included. But it was, so i'll fix it